app-containers/runc/runc-1.3.3-r1.ebuild

```diff
@@ -19,15 +17,14 @@ S="${WORKDIR}/${PN}-${MY_PV}"
 
 LICENSE="Apache-2.0 BSD-2 BSD MIT"
 SLOT="0"
-KEYWORDS="amd64 ~arm ~arm64 ppc64 ~riscv ~x86"
+KEYWORDS="~amd64 ~arm ~arm64 ~ppc64 ~riscv ~x86"
 IUSE="apparmor hardened +kmem +seccomp selinux test"
 
 COMMON_DEPEND="
 	apparmor? ( sys-libs/libapparmor )
 	seccomp? ( sys-libs/libseccomp )"
 DEPEND="${COMMON_DEPEND}"
 RDEPEND="${COMMON_DEPEND}
-	!app-emulation/docker-runc
```

Comment:
Just keep this please, the check is kind of buggy in pkgcheck. Just make the config changes instead.

Reply:
Sure, reverted.